internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,70 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
+	excludeDirs := []string{".git", ".zeabur"}
+	err = deleteFilesInDirectory(excludeFiles, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete files in directory: %w", err)
+	}
+	err = deleteDirectoriesInDirectory(excludeDirs, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete directories in directory: %w", err)
+	}
+
+	return nil
+}
+
+func deleteFilesInDirectory(deleteFiles []string, path string) error {
+	// walk through the directory
+	err := filepath.Walk(path, func(filePath string, fileInfo os.FileInfo, err error) error {
```

Comment:
Do we really need to delete recursively?

Reply:
Because these exclude_files are unnecessary or vulnerable. We need to exclude them in case the user forget to ignore them.

Reply:
Okay. Note that it may goes too deep that may introduce Denial of Service in builder, though it is rare to happen.